internal/scaffold/ansible/watches_test.go

```diff
@@ -0,0 +1,127 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ansible
+
+import (
+	"fmt"
+	"io/ioutil"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/operator-framework/operator-sdk/internal/scaffold"
+	"github.com/operator-framework/operator-sdk/internal/util/fileutil"
+)
+
+func TestUpdateAnsibleWatchForResource(t *testing.T) {
+
+	watchFilePath := "./testdata/valid/watches.yaml"
+	if err := ioutil.WriteFile(watchFilePath, []byte(sampleRole), fileutil.DefaultFileMode); err != nil {
+		fmt.Printf("failed to update %v", err)
+	}
```

Comment:
Ditto from question in Helm scaffold: Check this file in or make sure it is cleaned up with `defer`

Reply:
Please see my [comment](https://github.com/operator-framework/operator-sdk/pull/2703#discussion_r405509376) for the cleanup question.